mkosi.images/base/mkosi.extra/usr/share/mangos/self_test.sh

```diff
@@ -12,17 +12,95 @@ systemctl is-active systemd-cryptsetup@var.service
 systemctl is-active systemd-cryptsetup@var\\x2dtmp.service
 mangosctl bootstrap
 mangosctl sudo enroll -g{vault-server,{nomad,consul}-{server,client}}s 127.0.0.1
-mangosctl sudo -- nomad job run /usr/share/mangos/test.nomad
-tries=10
-while ! mangosctl sudo -- nomad alloc logs -namespace=admin -task server -job test | grep SUCCESS
+mangosctl sudo -- nomad job run -detach /usr/share/mangos/test.nomad
+
+echo "Waiting for job allocation to start..."
+echo "Current time: $(date)"
+tries=60
+success=0
+
+# Temporarily disable exit-on-error for polling loop
+set +e
```

Comment:
This seems heavy handed. I assume it's for the `alloc_status=$(...)` call? Can you just put `|| true` after it?

Reply:
I just did this see whats next when it fail, will remove it when build is passing